README.md

```diff
@@ -0,0 +1,516 @@
+
+You can also define your own operations; the only requirement is that
+your operation should be a callable object expecting a `pillow.Image` as
+its first parameter and it should return the transformed image:
+
+```python
+def crop_face(image):
+    """Detect face in image and crop around it."""
+    # TODO: Fancy algorithm.
+    return image
+
+IMAGE_PROCESS = {
+    "face-thumbnail": [crop_face, "scale_out 150 150 True"]
+}
+```
+
+### Additional Settings
+
+#### Destination Directory
+
+By default, the new images will be stored in a directory named
+`derivative/<TRANSFORMATION_NAME>` in the output folder at
+the same location as the original image.
+For example if the original image is located in
+the `content/images` folder. The computed images will be stored
+in the `output/images/derivative/<TRANSFORMATION_NAME>`.
+All the transformations are done in the output directory in order
+to avoid confusion with the source files or if we test multiple
+transformations.
+You can replace `derivative` by something else using the
+`IMAGE_PROCESS_DIR` setting in your Pelican configuration file:
+
+```python
+IMAGE_PROCESS_DIR = "derivees"
+```
+
+#### Force Image Processing
+
+If the transformed image already exists and is newer than the original
+image, the plugin assumes that it should not recompute it again. You
+can force the plugin to recompute all images by setting
+`IMAGE_PROCESS_FORCE` to `True` in your Pelican configuration
+file.
+
+```python
+IMAGE_PROCESS_FORCE = True
+```
+
+#### Selecting a HTML Parser
+
+You may select the HTML parser which is used. The default is the builtin
+`html.parser` but you may also select `html5lib` or `lxml` by setting
+`IMAGE_PROCESS_PARSER` in your pelican configuration file , e.g.:
+
+```python
+IMAGE_PROCESS_PARSER = "html5lib"
+```
+
+For details, refer to the [BeautifulSoup documentation on
+parsers](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-a-parser).
+
+#### File Encoding
+
+You may select a different file encoding to be used by BeautifulSoup as it
+opens your files. The default is `utf-8`.
+
+```python
+IMAGE_PROCESS_ENCODING = "utf-8"
+```
+
+## Known Issues
+
+* Pillow, when resizing animated GIF files, does not return an animated file
+* test mostly pass, but not entirely. The tests also fail on Windows due to
+  path separator issues. The test suite remains a work in progress.
+
+## Contributing
+
+Contributions are welcome and much appreciated. Every little bit helps. You can contribute by improving the documentation, adding missing features, and fixing bugs. You can also help out by reviewing and commenting on [existing issues][].
+
+To start contributing to this plugin, review the [Contributing to Pelican][] documentation, beginning with the **Contributing Code** section.
+
+[existing issues]: https://github.com/pelican-plugins/image-process/issues
+[Contributing to Pelican]: https://docs.getpelican.com/en/latest/contribute.html
```

Comment:
Normally I'd agree, but it's such a short section. Otherwise they have to jump to CONTRIBUTING, only to find the info isn't actually there and that they have to make another jump to the docs hosted in the Pelican core repo.

Reply:
OK, I'll keep it that way, in line with the cookiecutter template.